compiler/src/dmd/link.d

```diff
@@ -958,50 +959,58 @@ version (Windows)
 
 /***************************************
  * Run the compiled program.
- * Return exit status.
+ * Params:
+ *      exefile = program name
+ *      verbose = print command to be executed
+ *      eSink = message sink
```

Comment:
missing `runargs` and `verbose` in documentation

Reply:
fixed